src/routes/auth/login.ts

```diff
@@ -1,23 +1,27 @@
 import jws from 'jws'
 export async function login (req, res): Promise<void> {
-  const username = req.body.username
-  const password = req.body.password
-  if (username && password) {
+  if (req.mpsService.config.jwt_secret && req.mpsService.config.web_admin_user && req.mpsService.config.web_admin_password) {
+    const username = req.body.username
+    const password = req.body.password
+    if (username && password) {
```

Comment:
shouldn't need this check, should be using `express-validator` for this

Reply:
Will add a validator. Shall I use the same password constraints as of mps password  which is limited to length of 16?

Reply:
done